src/components/ImageView/index.js

```diff
@@ -116,6 +120,14 @@ class ImageView extends PureComponent {
         }
     }
 
+    onMouseUp(e) {
```

Comment:
```suggestion
    trackPointerPosition(e) {
```

I feel we can use this as the name.